backend/apid/routers/routers.go

```diff
@@ -95,6 +95,10 @@ func HTTPStatusFromCode(code actions.ErrCode) int {
 		return http.StatusConflict
 	case actions.PaymentRequired:
 		return http.StatusPaymentRequired
+	case actions.PermissionDenied:
+		return http.StatusForbidden
```

Comment:
Should we return http.StatusNotFound instead here?

Reply:
Excellent point!